Brokerages/QuantConnect.Brokerages.csproj

```diff
@@ -45,96 +45,92 @@
     <Reference Include="CSharpAPI">
       <HintPath>InteractiveBrokers\CSharpAPI.dll</HintPath>
     </Reference>
-    <Reference Include="IKVM.OpenJDK.Beans, Version=7.2.4630.5, Culture=neutral, PublicKeyToken=13235d27fcbfff58, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\IKVM-WithExes.7.3.4830.1\tools\IKVM.OpenJDK.Beans.dll</HintPath>
+    <Reference Include="IKVM.AWT.WinForms, Version=8.1.5717.0, Culture=neutral, PublicKeyToken=13235d27fcbfff58, processorArchitecture=MSIL">
```

Comment:
I presume these are the 'latest' versions that you manually converted? I forgot that FXCM connected via java code. Just wondering how we can document what was done here to produce these binaries so someone can follow a similar process in the future. Perhaps adding some details to the commit message would be sufficient

Reply:
See https://github.com/QuantConnect/FXCM/pull/1 for update to FXCM libraries.

Reply:
The only extra step required (after building a new version of `QuantConnect.Fxcm.dll` in https://github.com/QuantConnect/FXCM) is to manually copy the DLL in the `Brokerages/Fxcm` folder and push it to the LEAN repo (as done in this PR). 
Maybe in the future, we could create a NuGet package for it.